### PR TITLE
Close published form to stop responses

### LINKED ISSUE #13: As an Admin, I can close the form in order to stop the responses.
## Description
As an Admin, I can close the form in order to stop the responses.

## Mocks
n/a

### Update [Required]
- [ ] Button to close the form
- [ ] Connect API to set the allowed responses field to False

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1-2 days

### PR DESCRIPTION
**Pull request by bismitaguha**
### Description

- Added a button to close a published form
- Only admins can perform the function

Fixes #13 

### Type of Change:

- Code
- User Interface

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?

N/A

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] Any dependent changes have been merged

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by bismitaguha** [comment]
Only the last commit is relevant to this PR.
@sidvenu @abha224

**Review by sidvenu — CHANGES_REQUESTED** [review]
As said in the OSP weekly meeting, please add a message saying "The form is no longer accepting responses" if the form is closed, rather than displaying "Form not found" in all cases.

File: src/components/PublishedForm.js [hunk 1]
```diff
@@ -71,14 +71,14 @@ class PublishedForm extends Component {
 
     unpublish = (e, id) => {
         const data = {
-            published_status: 'False'
+            published_status: 'unpublished'
```
Comment (abha224) [inline src/components/PublishedForm.js]: @bismitaguha You have used the publish_status variable to denote unpublished, published and closed. Can you use a different variable for denoting status= closed? something like form_status?

**Comment by abha224** [comment]
@bismitaguha Pls add a screenshot/gif whenever you make a pr. Its easier to understand the change that way.

File: src/components/PublishedForm.js (hunk 1 above)
Comment (bismitaguha) [reply src/components/PublishedForm.js]: This is the variable present in the backend also, I cannot change it.

Comment (sidvenu) [reply src/components/PublishedForm.js]: @abha224 I believe having an enum of the 3 statuses (i.e., whatever changes Bismita has made) would make more sense than having true or false for two separate parameters.

**Review by abha224 — APPROVED** [review]

**Review by sidvenu — APPROVED** [review]

**Pull request merged**